compiler/src/dmd/id.d

```diff
@@ -531,6 +531,9 @@ immutable Msgtable[] msgtable =
     { "udaMustUse", "mustuse" },
     { "udaStandalone", "standalone" },
 
+    // Editions
+    { "__experimental_edition_latest", },
```

Comment:
I'd be fine with approving/merging right now if it had `_do_not_use` in the name.